contrib/export_xivo_2018.sh

```diff
@@ -0,0 +1,407 @@
+#!/bin/bash
+# Copyright 2021 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+set -e
+set -u  # fail if variable is undefined
+set -o pipefail  # fail if command before pipe fails
+
+# This script should be executed as user postgres
```

Comment:
Could you add a comment describing the difference between the export_xivo_2018 and export_xivo scripts? To be able to spot the differences between the 2 versions easily when/if we are going to modify the scripts

Reply:
mainly some useful debug information and cleaning SQL queries

Reply:
Should we remove the old version then?

Reply:
i will do another PR, but i prefer to keep it for the moment